lib/yggdrasil/player.ex

```diff
@@ -12,9 +13,16 @@ defmodule Yggdrasil.Player do
     Supervisor.add_player user_id, [owner_pid, push_msg]
   end
 
-  def run_cmd(user_id, cmd = %Message{type: :command}) do
-    player_pid = Registry.get_player user_id
-    GenServer.cast(player_pid, {:run_cmd, cmd})
+  def run_cmd(user_id, cmd) do
+    # horrible terrible temporary implementation
+    # room would eventually do all of this
+    ctxt = %Yggdrasil.Room.Context{ player: user_id }
```

Comment:
Ah yeah, totally agree it's going to look awkward until we have more components. 

I think this is a great start and gives us commands so I'm cool with that. Right now I think what you have we can't really represent any better than you have it -- we lack a room etc and it puts a hook in for context which I think is good.